### PR TITLE
Update build_image.yml

### PR DESCRIPTION
**Pull request by arnasbr**
Need to generate docker images from branch

### CODE REVIEW

File: .github/workflows/build_image.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: Docker Image
 on:
   push:
     branches: [ "master" ]
+  workflow_dispatch:
```
Comment (gergelytraveltime) [inline .github/workflows/build_image.yml:6]: is this a valid yaml?

Comment (arnasbr) [reply .github/workflows/build_image.yml:6]: it should be, worst case I'll make another PR

Comment (gergelytraveltime) [reply .github/workflows/build_image.yml:6]: I think it's not

Comment (arnasbr) [reply .github/workflows/build_image.yml:6]: Yaml validator passes. I'll see if I can test the action itself before merging

Comment (arnasbr) [reply .github/workflows/build_image.yml:6]: https://github.com/traveltime-dev/web-dev/blob/fcadea8fad62dc5d8a0e4470fdbe4a46fa565800/.github/workflows/demo-apps-update-dev.yaml#L4 same thing is done here

**Comment by danielnaumau** [comment]
Why not to do it locally?

**Comment by arnasbr** [comment]
> Why not to do it locally?

Just thought this could be useful. But tbh since this is a public repo, might not be the best idea to do this. I'm scrapping this PR

**Pull request closed without merging**